winml/test/common/test.h

```diff
@@ -25,3 +25,9 @@ constexpr bool alwaysFalse() {
 #include "taefTestMacros.h"
 #endif
 #endif
+
+#if defined(BUILD_GOOGLE_TEST) || defined(BUILD_TAEF_TEST)
```

Comment:
One of them should always be defined, right?

Reply:
true, but I felt like being explicit is good. That being said perhaps I should be explicit by throwing an exception if neither are defined?

Reply:
I think we can pick googletest if neither is defined... I was thinking about it for a while, it also helps if you're using intellisense in VS Code (currently it doesn't autocomplete or locate any of the test macros because they are behind the #ifdefs, picking googletest if neither is defined would solve it)

Reply:
Actually, I'll go ahead and always define the SkipTest.

Reply:
alright I went ahead and set it to default to googletest

Reply:
but this causes SkipTest to be defined twice,  once in winml_test_common and once in each winml_test_api for example, which causes a build error. It seems like winml_test_common doesn't even need to build these files since all the implementation is in the headers which are being included in the test binaries directly. What do you think?